Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.3
```

Comment:
this file cannot be versionned in our dev repo and deployed each time ? (like the podfile for instance)

Reply:
This file is a placeholder, it will be regenerated by our script like the `podspec`, you'll see more details in here https://github.com/ContentSquare/cs-ios-sdk/pull/1224